test/availability/strncasecmp.c

```diff
@@ -1,6 +1,13 @@
 #include "yarp/defines.h"
+#ifndef _WIN32
 #include <strings.h>
+#endif
 
 int main() {
+#if defined(_WIN32)
+    return 1;
```

Comment:
I'm assuming this means that `strncasecmp` is always available on windows no matter what?

Reply:
I had done this wrong, fixed it now to only include `strings.h` if we're not in windows. `strings.h` and relatedly `strncasecmp` aren't in windows!